src/sdk.js

```diff
@@ -800,5 +801,8 @@ export default class SharetribeSdk {
 
     // Assign SDK functions to 'this'
     sdkFns.forEach(({ path, fn }) => _.set(this, path, fn));
+
+    // Assign util functions to 'this'
+    _.set(this, 'util', util);
```

Comment:
Why are the utils defined as instance methods and not static like the Transit helpers? See:

https://github.com/sharetribe/flex-sdk-js/blob/master/src/index.js

Reply:
This was more like a convenience choice, in FTW sdk is injected to thunk functions in the duck files. There the sdk instance is at hand. Naturally it could also be a static method but when looking at the use in context of the duck files, having it in the instance felt more natural.